src/components/WebexMeetingControl/WebexMeetingControls.js

```diff
@@ -10,15 +15,28 @@ export const MeetingContext = React.createContext();
  * @param {object} props
  * @returns {object} JSX of the component
  */
-export default function WebexMeetingControls({meetingID, children}) {
+export default function WebexMeetingControls({children, className, meetingID}) {
+  const mainClasses = {
+    [`${WEBEX_COMPONENTS_CLASS_PREFIX}-meeting-controls`]: true,
+  };
+
+  if (className) {
```

Comment:
Like in my other comment, this check is not necessary, and could be shortened by just adding `className` as a last item in `mainClasses`